### PR TITLE
Mood graph, AI tips based on entries, and new UI

### PR DESCRIPTION
**Pull request by kparajuli21**
https://github.com/user-attachments/assets/c13e5562-c664-4b3a-a885-2beef971aef6

### CODE REVIEW

File: VitalTrackServer/activity/api_log_entry.py [hunk 1]
```diff
@@ -1,35 +1,58 @@
...
-            activity=entry_data.get("activity"),
+            stress = entry_data.get("stress"),
+            activity=entry_data.get("activity",[]),
             symptoms=entry_data.get("symptoms", [])
+            
         )
+        print(entry)
 
         user = mongo_db_facade.get_user_by_id(user_id)
+        print("USER ID: ",user.user_id)
```
Comment (MerwinAamir1) [inline VitalTrackServer/activity/api_log_entry.py:38]: looks like you forgot to remove these (not blocking you on these tho)

File: VitalTrackServer/activity/api_log_entry.py [hunk 2]
```diff
@@ -1,35 +1,58 @@
...
 from flask import Blueprint, jsonify, request
 from remotecalls.mongodb_facade import mongo_db_facade
 from model.entries_model import Entry
+from dateutil import parser
 
 log_entry_blueprint = Blueprint("logEntry", __name__)
 
 @log_entry_blueprint.route("/logEntry", methods=["POST"])
 def log_entry():
+    print("log_entry route was hit")  # Check if this gets printed
```
Comment (MerwinAamir1) [inline VitalTrackServer/activity/api_log_entry.py:12]: same as below

File: VitalTrackApp/src/screens/Analytics.js [hunk 3]
```diff
@@ -1,31 +1,61 @@
...
-        <Text style={styles.title}>Analytics Page</Text>
+        <View style={styles.header}>
+            <Text style={styles.headerText}>Analytics</Text>
+          </View>
         <Text style={styles.subtitle}>An overview of your wellbeing</Text>
 
         <View style={styles.section}>
             <Text style={styles.title}>Insights</Text>
-            <Text style={styles.subtitle}>You felt [less stressed] when you [did yoga] this week.</Text>
+            <Text style={styles.caption}>{tips ? tips.message : "Loading tips..."}</Text>
```
Comment (MerwinAamir1) [inline VitalTrackApp/src/screens/Analytics.js:43]: when we render these tips or anything in the analytics page can we use the TextGeneratorEffect I created. You can simply Wrap it like this <TextGeneratorEffect text={tips}/>

**Review by MerwinAamir1 — APPROVED** [review]

File: VitalTrackApp/src/screens/Analytics.js (hunk 3 above)
Comment (kparajuli21) [reply VitalTrackApp/src/screens/Analytics.js]: Okay, I changed it to use text generator effect

**Pull request merged**